plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -119,18 +116,13 @@ export const createPublishGithubPullRequestAction = ({
     id: 'publish:github:pull-request',
     schema: {
       input: {
-        required: ['owner', 'repo', 'title', 'description', 'branchName'],
+        required: ['repoUrl', 'title', 'description', 'branchName'],
```

Comment:
Clearly the old version of this is an error - it should not have required any of them and used a `oneOf` prop to specify either owner/repo/host or repoUrl. Sorry about that 🙈